cypress/e2e/08-account_creation.cy.js

```diff
@@ -62,162 +61,63 @@ Cypress.Commands.add("verifySecurityCode", (code) => {
   cy.get(submitButton).first().click();
 });
 
-describe("Okta account creation", () => {
+describe("Checks each of the account creation MFA options", () => {
```

Comment:
refactored this file for a similar reason to [7397](https://github.com/CDCgov/prime-simplereport/pull/7397), namely the new within-file test isolation features introduced in v12.